### ISSUE TITLE
For Cli How can I pass and use parameters?

### ISSUE THREAD

**Issue by chenghuang-mdsol** [kurtmkurtm/LiquidTestReports#30]

**Comment by kurtmkurtm** [comment]
Hi, do you mean passing additional parameters to use in your own template?

**Comment by chenghuang-mdsol** [comment]
> Hi, do you mean passing additional parameters to use in your own template?

Yes. for cli, using custom template

**Comment by kurtmkurtm** [comment]
It wasn't possible using the current implementation to pass custom parameters.

In the interests of keeping this feature across the CLI and Test Logger implementations, I am adding this feature in #31

**Comment by kurtmkurtm** [comment]
Feature published in 1.4.3-beta, there's some details in the readme, but here is a sample from the scripts.

``` pwsh
liquid `
--inputs "File=**/*sample.trx;Folder=../Test/LiquidTestReports.Cli.Tests/TrxTestInput;Format=Trx;RunId=123" `
--template "../Test/LiquidTestReports.Cli.Tests/TemplateTestInput/example.md" `
--parameters "Environment=UAT;TicketId=abc123" `
--output-file custom.md  
```

Closing issue with this update
